### DISCUSSION TITLE
Microsoft.IdentityModel.Protocols.OpenIdConnect 7.1.2

### DISCUSSION THREAD

**Discussion by cheng93** [AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet#2441] [General]
I'm starting migrate projects to net8.0 but I just hit a blocker. 

Microsoft.AspNetCore.Authentication.JwtBearer 8.0.1
Microsoft.AspNetCore.Authentication.OpenIdConnect 8.0.1

depends on Microsoft.IdentityModel.Protocols.OpenIdConnect 7.1.2

but it's not on nuget yet

https://www.nuget.org/packages/Microsoft.IdentityModel.Protocols.OpenIdConnect/#versions-body-tab

**Comment by martincostello** [comment]
These new package versions are now in NuGet.org.
